admin-commands/reaction-roles.md

```diff
@@ -4,54 +4,44 @@
 
 This command is used to manage all settings for reaction roles.
 
+{% hint style="info" %}
 
 This page covers in-depth technical details about the command. For a start-up guide on reaction roles, see [Setting up reaction roles](../start-up/setting-up-reaction-roles.md).
-
+{% endhint %}
 
 ### Subcommands
 
-#### Config
-
-This command creates a new group, given a name. A server can have a maximum of 10 groups.
-
-The only required argument for this option is `group`, which will later be used to identify your group.
-
 #### Set
 
 This command adds pairs to the group (a pair being an emoji connect to a role). A group can have a maximum of 20 pairs.
 
-The required arguments for this option are `group`, `emoji`, and `role`. `Group` is the group that you want to apply pairs to and `emoji` is the emoji you want the user to react with to toggle the `role`.
-
+![](<../.gitbook/assets/ReactionRoleSetup2.png>)
 
-#### Apply
-
-Applies the selected group to a given message. Instructions on how to do this are given [here](../start-up/setting-up-reaction-roles.md#applying-your-group-to-messages).
-
-More than one group can be applied to a same message at any time. The same group can be applied to a maximum of 5 different messages.
-
-#### Unapply
-
-Has the opposite effect of `apply`, ergo, it will remove the group from the specified message. 
+The required arguments for this option are `group`, `emoji`, and `role`. `Group` is the group that you want to apply pairs to and `emoji` is the emoji you want the user to react with to toggle the `role`.
 
 #### Delete
 
-This deletes a given group(`group` is the required argument.).
+This deletes a given group (`group` is the required argument.).
 
+{% hint style="danger" %}
 This action is irreversible. You will be allowed to create a new group with the same name as a deleted group.
+{% endhint %}
 
 #### Info
 
 Takes a group name as required argument, and shows some information, such as the emoji/role couples and settings for the group.
 
+{% hint style="info" %}
 You can edit group options with the `config` command
+{% endhint %}
 
 #### List
 
 This option lists all the available groups by name and amount of roles set. No arguments required.
 
 #### Autorepair
 
-This removes any deleted emoji/role from the group(the only required argument), cleaning things up. It will not search for deleted messages.
+This removes any deleted emoji/role from the group (the only required argument), cleaning things up. It will not search for deleted messages.
```

Comment:
There's an optional argument that will unapply all messages from the group, to be used with caution

(this is meant in case a group reaches the maximum of 5 messages but they cannot be unapplied individually if they're deleted)